auto_nag/scripts/b2g_query_creator.py

```diff
@@ -104,11 +105,15 @@ def createQueriesList(print_all, queries_dir):
 ]
 
 
-def cleanUp():
-    for file in os.listdir(queries_dir):
-        if file.startswith(str(datetime.date.today())):
-            os.remove(os.path.join(queries_dir, file))
-
+def cleanUp(queries_dir):
+    try:
+        for file in os.listdir(queries_dir):
+            if file.startswith(str(datetime.date.today())):
```

Comment:
what do we store in it?
Anyway, we could clear that stuff in the test itself, don't you think?